OneCommunityGlobal/HighestGoodNetworkApp
# Changhao usersearchpanel unit test

# Description
added new unit tests for usersearchpanel.jsx

## How to test:
1. check into current branch
2. do `npm install` and `npm test UserSearchPanel.test.js` to run this PR locally
3. confirm all 12 tests are passed

## Screenshots:
[
<img width="677" alt="Screenshot 2023-12-08 at 8 24 45 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104778270/3227f530-f873-44ba-a47f-3fa56d9015d5">
](url)
